src/boxrobots/src/visualize.jl

```diff
@@ -90,45 +90,23 @@ function draw_environment(vis::DrakeVisualizer.Visualizer, env::Environment)
   end
 end
 
-function playback_trajectory(vis::DrakeVisualizer.Visualizer, traj::Trajectory{BoxRobotSimulationData}; options=nothing, playback_speed=1.0)
+function playback_trajectory{SimData<:BoxRobotSimulationData}(vis::DrakeVisualizer.Visualizer, traj::Trajectory{SimData}; options=BoxRobotVisualizerOptions())
```

Comment:
BTW, the `options=BoxRobotVisualizerOptions()` above is the preferred way of doing defaults. The way you were doing it makes `options` type unstable (since it can be either of type `Void` for `nothing` or of type `BoxRobotVisualizerOptions`). `BoxRobotVisualizerOptions()` will not be run if you do pass `options`, so there's no overhead there.